README.md

```diff
@@ -106,18 +106,24 @@ FHIR Operation to obtain a detailed set of FHIR resources of diverse resource ty
 
 Endpoint: `GET [fhir base]/Patient/$export`
 
+Alternatively, a POST request (`POST [fhir base]/Patient/$export`) can be sent. The export parameters should be supplied using a FHIR [Parameters Resource](http://hl7.org/fhir/R4/parameters.html) in the request body.
```

Comment:
```suggestion
Alternatively, a POST request (`POST [fhir base]/Patient/$export`) can be sent. The export parameters must be supplied using a FHIR [Parameters Resource](http://hl7.org/fhir/R4/parameters.html) in the request body.
```
Maybe change this since if it isn't supplied using a parameters resource then an error is thrown.